portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -9,10 +9,29 @@
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
+    private int pageViews = 0;
```

Comment:
[4.2](https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation): Each time a new block or block-like construct is opened, the indent increases by two spaces.
```suggestion
  private int pageViews = 0;
```